planner/core/explain.go

```diff
@@ -683,21 +680,28 @@ func (p *LogicalIndexScan) ExplainInfo() string {
 			}
 		}
 	}
-	if len(p.AccessConds) > 0 {
-		fmt.Fprintf(buffer, ", cond:%v", p.AccessConds)
+	if len(s.AccessConds) > 0 {
+		fmt.Fprintf(buffer, ", cond:%v", s.AccessConds)
 	}
 	return buffer.String()
 }
 
 // ExplainInfo implements Plan interface.
-func (p *TiKVSingleGather) ExplainInfo() string {
-	buffer := bytes.NewBufferString(p.Source.ExplainInfo())
-	if p.IsIndexGather {
-		buffer.WriteString(", index:" + p.Index.Name.String())
+func (sg *TiKVSingleGather) ExplainInfo() string {
+	buffer := bytes.NewBufferString(sg.Source.ExplainInfo())
+	if sg.IsIndexGather {
+		buffer.WriteString(", index:" + sg.Index.Name.String())
```

Comment:
should it be:
```suggestion
		buffer.WriteString("index:" + sg.Index.Name.String())
```

Reply:
Are you sure to do the change? It will cause explain like:
```
"    TiKVSingleGather_9 input:[Group#5], table:tindex:c_d_e"
```